client/components/tech-update-form.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Form, Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function TechUpdateForm({ project, onSave, onCancel }) {
+  const poNumber = project.poNumber;
```

Comment:
You could grab all of these at once with destructuring:
`const { poNumber, name, etc. } = project;`